crates/types/build.rs

```diff
@@ -0,0 +1,19 @@
+fn main() {
+    use std::{fs::OpenOptions, path::Path};
+
+    // Only re-run if the Minecraft JSON data files were modified.
+    //println!("cargo:rerun-if-changed=../../mc-data/blocks.json");
+
+    let out_dir = std::env::var_os("OUT_DIR").expect("environment variable OUT_DIR set");
+    let mut dest_file = OpenOptions::new()
+        .write(true)
+        .create(true)
+        .truncate(true)
+        .open(Path::new(&out_dir).join("blocks.rs"))
+        .expect("create destination file");
+
+    let src_path = Path::new("../../contrib/blocks.json");
```

Comment:
Make this configurable through an env variable

Reply:
Done; see [d8fa342:/crates/data-generator/Cargo.toml](https://github.com/rjacraft/rjacraft/blob/d8fa342630650f9e78acc4219e82bd18e281f618/crates/types/build.rs#L12).